src/main/java/ca/corefacility/bioinformatics/irida/config/services/IridaApiServicesConfig.java

```diff
@@ -418,5 +428,20 @@ public SpringTemplateEngine exportUploadTemplateEngine() {
 	public NreplServerSpring nRepl() {
```

Comment:
I just see this here and it is not related, but I thought we got rid of the REPL stuff?